Sources/Exporters/Prometheus/PrometheusMetric.swift

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+struct PrometheusMetric {
+    let contentType = "text/plain; version = 0.0.4"
+
+    static let firstCharacterNameCharset = CharacterSet(charactersIn: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_:")
+    static let nameCharset = CharacterSet(charactersIn: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_:")
+    static let firstCharacterLabelCharset = CharacterSet(charactersIn: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_")
+    static let labelCharset = CharacterSet(charactersIn: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_")
+
+    var values = [PrometheusValue]()
+    var name: String
+    var description: String
+    var type: String = ""
+
+    func write() -> String {
+        
+        var output = ""
+        
+        let name = PrometheusMetric.getSafeMetricName(name: self.name)
+        
+        if !description.isEmpty {
+            output += "# HELP "
+            output += name
+            output += PrometheusMetric.getSafeMetricDescription(description: description)
+            output += "\n"
+        }
+        
+        if !type.isEmpty {
+            output += "# TYPE "
+            output += name
+            output += " "
+            output += type
+            output += "\n"
+        }
+        
+        let now = String(Int64((Date().timeIntervalSince1970 * 1000.0).rounded()))
```

Comment:
should we have a single `now` value for all metrics in a single output?

Reply:
Yes, thats a very good idea, metrics are already collected and we wont be adding the time taken for writing them, I will add this change